openmmtools/multistate/sams.py

```diff
@@ -628,7 +648,6 @@ def _update_logZ_estimates(self, replicas_log_P_k):
         for (replica_index, state_index) in enumerate(self._replica_thermodynamic_states):
             logger.debug(' Replica %d state %d' % (replica_index, state_index))
             # Compute attenuation factor gamma
-            beta_factor = 0.8
```

Comment:
This may be the main cause of test errors. The local variable `beta_factor` is deleted here, but I don't see `self.beta_factor` used anywhere.